README.md

```diff
@@ -6,25 +6,37 @@ This is an example Solo.io Gloo Platform Dev Portal frontend app, built with [Vi
 
 ## Initial Setup
 
-1. Install [Node v16.14.2](https://nodejs.org/en/blog/release/v16.14.2) and [yarn](https://yarnpkg.com/).
-
-2. Run the following to download and initialize the latest commit of this repository's main branch.
+1. Run the following to download and initialize the latest commit of this repository's main branch.
 
    - This command uses [tmplr](https://github.com/loreanvictor/tmplr) for personalization of your developer portal. Currently, this just includes adding your company name, which shows up on the home page and on the website title, but could include other customization options in the future. The tmplr setup can be re-run at any time by running `npx tmplr` from the project's root directory.
 
    ```shell
    mkdir portal-test && cd portal-test && npx tmplr solo-io/dev-portal-starter#main
    ```
 
-3. Create a `.env.local` file in the `projects/ui` folder. Replace environment variable values to match your Gloo Platform Portal and oauth provider's installation. This file is ignored by git.
+2. Create a `.env.local` file in the `projects/ui` folder. Replace environment variable values to match your Gloo Platform Portal and oauth provider's installation. This file is ignored by git.
 
    ```shell
    VITE_PORTAL_SERVER_URL="/v1"
    VITE_CLIENT_ID="your-client-id"
    VITE_TOKEN_ENDPOINT="your-token-endpoint"
+   VITE_AUTH_ENDPOINT="your-auth-endpoint"
+   VITE_LOGOUT_ENDPOINT="your-logout-endpoint"
+   ```
+
+   <details><summary>Sample values for Keycloak</summary>
+
+   ```shell
+   VITE_PORTAL_SERVER_URL="/v1"
+   VITE_CLIENT_ID="your-client-id"   # the client registered in the Auth Server
+   VITE_TOKEN_ENDPOINT="https://${KEYCLOAK_URL}/auth/realms/master/protocol/openid-connect/token"
```

Comment:
It's `https://${KEYCLOAK_URL}/auth/realms/master/protocol/openid-connect/token` on Keycloak 12.0.4, but on the latest version it's `https://${KEYCLOAK_URL}/realms/master/protocol/openid-connect/token`. Not sure if we need to do anything about that here, but came across it when I was testing both versions.